src/domain.rs

```diff
@@ -304,13 +314,17 @@ impl<F> Domain<F> {
         unsafe { self.untagged[Self::calc_shard(retired)].push(retired, retired) };
         self.count.fetch_add(1, Ordering::Release);
 
+        #[cfg(not(feature = "std"))]
+        return 0;
+        #[cfg(feature = "std")]
         self.check_threshold_and_reclaim()
     }
 
     fn threshold(&self) -> isize {
         RCOUNT_THRESHOLD.max(HCOUNT_MULTIPLIER * self.hazptrs.count.load(Ordering::Acquire))
     }
 
+    #[cfg(feature = "std")]
```

Comment:
Hmm, I think this is still fine without `std` — it's just the `due_time` part that needs to be skipped.

Reply:
Woops